src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp

```diff
@@ -385,7 +385,7 @@ void LightPathsWidget::render_light_path(const size_t light_path_index) const
         light_path_recorder.get_light_path_vertex(i, v1);
 
         auto radiance = Color3f::from_array(v1.m_radiance);
-        radiance /= sum_value(radiance);
+        radiance /= radiance + Color3f(1.0);
```

Comment:
Possibly clamp this value, such that it doesn't get too low (and thus won't be seeable anymore).

Reply:
Good observation, however I fear that any non-zero value may misrepresent the fact that the path effectively carries zero radiance. I feel like we're better off with black rays which anyway will be visible on non-black background.